<a href="https://colab.research.google.com/github/JosephHobbs9292/Project-Two/blob/main/Project_2_Data(2).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
## Importing Libraries

import pandas as pd

In [2]:
## Mounting Drive
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [6]:
df = pd.read_csv("/content/heart.csv")
df.head()

Age Sex ChestPainType  RestingBP  Cholesterol  FastingBS RestingECG  MaxHR  \
0   40   M           ATA        140          289          0     Normal    172   
1   49   F           NAP        160          180          0     Normal    156   
2   37   M           ATA        130          283          0         ST     98   
3   48   F           ASY        138          214          0     Normal    108   
4   54   M           NAP        150          195          0     Normal    122   

  ExerciseAngina  Oldpeak ST_Slope  HeartDisease  
0              N      0.0       Up             0  
1              N      1.0     Flat             1  
2              N      0.0       Up             0  
3              Y      1.5     Flat             1  
4              N      0.0       Up             0

In [7]:
df.shape

(918, 12)

---
## Data Explanation
---

1. Kaggle.com
2. The data shows biomedical stats related to people who had cardiac incidents. 
3. The target column is heart disease
4. Each row is one person and their related medical data
5. This is a binary classification problem focusing on wether or not a person has heart disease
6. There are eleven features. 
7. 918
8. None!